pkg/provision/engine.go

```diff
@@ -1162,3 +1189,57 @@ func (e *NativeEngine) GetWorkloadStatus(id string) (gridtypes.ResultState, bool
 
 	return wl.Result.State, true, nil
 }
+
+// getTwinVerificationStatus make sure the account used is verified we have the user public key in bytes(pkBytes)
+func getTwinVerificationStatus(twinID uint32) (status string) {
+	status = "FAILED"
```

Comment:
Please use a const instead. If possible values can only be success/failure then it's better to use a boolean maybe

Define a new type alias so it's not confused with actual strings